docs/kafka/kafka-bin-scripts/README.adoc

```diff
@@ -207,13 +211,13 @@ You can use the `kafka-console-consumer` script to consume messages from Kafka t
 * You used the `kafka-console-producer` script to produce example messages to a topic.
 
 .Procedure
+. Open a second terminal window or tab, separate from the producer.
 
-. On the command line in a separate terminal from your producer, enter the following command to start the `kafka-console-consumer` script.
+. On the command line, enter the following command to start the `kafka-console-consumer` script.
```

Comment:
I would keep the phrase about using a separate terminal. The reason is that while the producer is running, you can't run the consumer in the same terminal session. Having the producer and consumer running side-by-side is helpful for seeing the messages go from the producer to the consumer.

Reply:
@bhardesty I added a step before that reads 'Open a second terminal window or tab, separate from the producer.' so that information is still there.